ci/python-publish.yml

```diff
@@ -1,4 +1,4 @@
-# This workflow will upload a Python Package using Twine when a release is created
+# This workflow will upload a Python Package to PyPI when a release is created
 # For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
```

Comment:
If it's not against some policy, let's link https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ — we're trying to keep it up-to-date over in PyPUG, which often happens sooner than GH gets docs updated.

Reply:
Makes sense to me. I'll defer to GH folks for any actual policy here 🙂